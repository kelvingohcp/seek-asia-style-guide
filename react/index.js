export { default as StyleGuideProvider } from './StyleGuideProvider/StyleGuideProvider';
export { default as SeekApp } from './StyleGuideProvider/StyleGuideProvider'; // Alias for backwards compatibility

// Components
export { default as Alert } from './Alert/Alert';
export { default as Button } from './Button/Button';
export { default as FilterButton } from './FilterButton/FilterButton';
export { default as ButtonGroup } from './ButtonGroup/ButtonGroup';
export { default as ListItem } from './ListItem/ListItem';
export { default as Loader } from './Loader/Loader';
export { default as Logo } from './Logo/Logo';
export { default as LogoRainbow } from './LogoRainbow/LogoRainbow';
export { default as Rating } from './Rating/Rating';
export { default as NavigationBar } from './NavigationBar/NavigationBar';
export { default as JobCard, trackLinkType } from './JobCard/JobCard';
export { default as JobCardLoader } from './JobCardLoader/JobCardLoader';
export { default as CardLoader } from './CardLoader/CardLoader';
export { default as CardLoadingBar } from './CardLoadingBar/CardLoadingBar';
export { default as Header } from './Header/Header';
export { default as ShowMore } from './ShowMore/ShowMore';
export { default as TitledCard } from './TitledCard/TitledCard';
export { default as Icon } from './Icon/Icon';
export { default as DropdownList } from './DropdownList/DropdownList';
export { default as Tag } from './Tag/Tag';
export { default as Badge } from './Badge/Badge';

// Typography
export { default as Critical } from './Critical/Critical';
export { default as Paragraph } from './Paragraph/Paragraph';
export { default as Positive } from './Positive/Positive';
export { default as Secondary } from './Secondary/Secondary';
export { default as Strong } from './Strong/Strong';
export { default as Text } from './Text/Text';
export { default as TextLink } from './TextLink/TextLink';

// Layout
export { default as AsidedLayout } from './AsidedLayout/AsidedLayout';
export { default as Card } from './Card/Card';
export { default as CardGroup } from './CardGroup/CardGroup';
export { default as Columns } from './Columns/Columns';
export { default as Hidden } from './Hidden/Hidden';
export { default as PageBlock } from './PageBlock/PageBlock';
export { default as Section } from './Section/Section';

// Icons
export { default as CriticalIcon } from './CriticalIcon/CriticalIcon';
export { default as CrossIcon } from './CrossIcon/CrossIcon';
export { default as ErrorIcon } from './ErrorIcon/ErrorIcon';
export { default as FieldOfStudyIcon } from './FieldOfStudyIcon/FieldOfStudyIcon';
export { default as FlagHKIcon } from './FlagHKIcon/FlagHKIcon';
export { default as FlagIDIcon } from './FlagIDIcon/FlagIDIcon';
export { default as FlagMYIcon } from './FlagMYIcon/FlagMYIcon';
export { default as FlagPHIcon } from './FlagPHIcon/FlagPHIcon';
export { default as FlagSGIcon } from './FlagSGIcon/FlagSGIcon';
export { default as FlagTHIcon } from './FlagTHIcon/FlagTHIcon';
export { default as FlagVNIcon } from './FlagVNIcon/FlagVNIcon';
export { default as HelpIcon } from './HelpIcon/HelpIcon';
export { default as InfoIcon } from './InfoIcon/InfoIcon';
export { default as JobFunctionIcon } from './JobFunctionIcon/JobFunctionIcon';
export { default as JobInvitationIcon } from './JobInvitationIcon/JobInvitationIcon';
export { default as MoreIcon } from './MoreIcon/MoreIcon';
export { default as ProfileIcon } from './ProfileIcon/ProfileIcon';
export { default as StarIcon } from './StarIcon/StarIcon';
export { default as TickCircleIcon } from './TickCircleIcon/TickCircleIcon';

// Fields
export { default as Autosuggest } from './Autosuggest/Autosuggest';
export { default as Checkbox } from './Checkbox/Checkbox';
export { default as ClearField } from './ClearField/ClearField';
export { default as Dropdown } from './Dropdown/Dropdown';
export { default as EmailField } from './EmailField/EmailField';
export { default as MonthPicker } from './MonthPicker/MonthPicker';
export { default as SlideToggle } from './SlideToggle/SlideToggle';
export { default as Textarea } from './Textarea/Textarea';
export { default as TextField } from './TextField/TextField';

// Accessibility
export { default as ScreenReaderOnly } from './ScreenReaderOnly/ScreenReaderOnly';
export { default as ScreenReaderSkipLink } from './ScreenReaderSkipLink/ScreenReaderSkipLink';
export { default as ScreenReaderSkipTarget } from './ScreenReaderSkipTarget/ScreenReaderSkipTarget';

// Constants
export { default as Constants } from './Constants/Constants';
