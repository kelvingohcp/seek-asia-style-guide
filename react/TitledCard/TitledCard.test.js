import React from 'react';
import { shallow } from 'enzyme';
import { TitledCard, Text } from 'seek-asia-style-guide/react';

describe('Titled Card: ', () => {
  const titledCard = (
    <TitledCard text="Job Requirement">
      <Text>
        Take time off and schedule a date with us. We've made our class schedule
        and timetable handy to access when you need it. Check our schedules
        online to get the latest updates or changes. If there's one date you can
        make that won't disappoint you, it's a date with us. Choose your city or
        select your nearest club to get full schedules and timetables for all
        classes available at Celebrity Fitness. For the latest updates, check
        this page often
      </Text>
    </TitledCard>
  );
  it('should render value', () => {
    const wrapper = shallow(titledCard);
    expect(wrapper).toMatchSnapshot();
  });
});
