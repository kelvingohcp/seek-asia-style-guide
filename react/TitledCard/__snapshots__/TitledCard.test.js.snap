// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titled Card:  should render value 1`] = `
<Card>
  <Section
    className=""
    header={false}
    pullout={false}
    slim={false}
  >
    <Text
      baseline={true}
      strong={true}
      yelling={true}
    >
      Job Requirement
    </Text>
    <hr
      className="line"
    />
    <Text
      baseline={true}
    >
      Take time off and schedule a date with us. We've made our class schedule and timetable handy to access when you need it. Check our schedules online to get the latest updates or changes. If there's one date you can make that won't disappoint you, it's a date with us. Choose your city or select your nearest club to get full schedules and timetables for all classes available at Celebrity Fitness. For the latest updates, check this page often
    </Text>
  </Section>
</Card>
`;
